public/API/API_GetGameList.php

```diff
@@ -5,6 +5,8 @@
  *    i : console id
  *    f : 1=only return games where NumAchievements > 0 (default: 0)
  *    h : 1=also return hashes (default: 0)
+ *    o : offset (default: 0)
+ *    c : count (default: 0)
```

Comment:
I believe the default values for these is null, not 0.

Recommend changing "default: 0" to "optional" to better imply this.

Reply:
I checked `getGamesListByDev` function and saw that 0 is the default for both values

```
function getGamesListByDev(
    ?User $dev,
    int $consoleID,
    ?array &$dataOut,
    int $sortBy,
    bool $ticketsFlag = false,
    ?int $filter = 0,
    int $offset = 0,
    int $count = 0,
    ?string $listType = null
)
```

Reply:
Right, but before it hits the helper function it isn't actually 0. `request()->query('o')` is going to return `null` if the value is not present.

Put another way: if I explicitly gave an HTTP endpoint which supported `count` and `offset` parameters 0, I'd expect it to return no results in an array while possibly informing me of the total number of results available.

Reply:
I see. Changed to `(optional)`